app/src/androidTest/java/com/github/se/cyrcle/di/mocks/MockOfflineParkingRepository.kt

```diff
@@ -121,4 +143,14 @@ class MockOfflineParkingRepository @Inject constructor() : OfflineParkingReposit
     if (report.uid == "") onFailure(Exception("Error adding report"))
     onSuccess(reports[0])
   }
+
+  override fun addImageReport(
+      report: ImageReport,
+      parking: String,
+      onSuccess: (ImageReport) -> Unit,
+      onFailure: (Exception) -> Unit
+  ) {
+    if (report.uid == "") onFailure(Exception("Error adding report"))
```

Comment:
It is just a detail but you could use `isEmpty()` here for the condition.

Reply:
it's how it is everywhere haha. Would you like me to change it everywhere?